_sass/_uswds-custom-styles.scss

```diff
@@ -122,6 +122,51 @@
   }
 }
 
+//
+// Microsite Nav
+// ==========================================================================
+.microsite-header {
+  .usa-navbar {
+    background-image: url('../../img/microsite-banner/18_0314_ST_microsite-banner.png');
+    background-size: 100% 120px;
+    background-repeat: no-repeat;
+    background-position: top center;
+    max-width: none;
+    padding-bottom: px-to-rem(120);
+    padding-left: 0;
+    padding-right: 0;
+
+    @media
+    (-webkit-min-device-pixel-ratio: 2),
+    (min-resolution: 192dpi) {
+      background-image: url('../../img/microsite-banner/18_0314_ST_microsite-banner@2x.png');
+    }
+  }
+
+  .usa-logo {
+    margin: 0;
+  }
+
+  .usa-nav {
+    background-color: color("white");
+  }
+
+  .usa-nav .usa-current {
+
+    // Border on current link
+    &::after {
+      border-radius: radius(0);
+    }
+  }
+
+  .usa-nav__secondary {
+    @include at-media($theme-header-min-width) {
+      bottom: px-to-rem(198);
+    }
+  }
+}
+
+
```

Comment:
2. The microsite navigation also uses a extended header, so I've modified our existing setup to avoid style conflicts. You'll see that the site-header has a separate class than the microsite-header. I tend to create a new class for every variant to make it easier to theme. But you should always try to standardize as best you can.